CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### ✅ Added
 - Introduce message localization. See [docs](https://getstream.io/chat/docs/ios-swift/translation/?language=swift) for more info [#1867](https://github.com/GetStream/stream-chat-swift/issues/1867)
 - Add support for multiple push providers [#1864](https://github.com/GetStream/stream-chat-swift/issues/1864)
+- Add support for  multiple active channel lists at the same time [#1879](https://github.com/GetStream/stream-chat-swift/pull/1879)
```

Comment:
⚠️  Don't forget to properly edit the changelog, after 4.13.0 release

Reply:
Good one!